src/config/homepage-config.interface.ts

```diff
@@ -19,4 +19,8 @@ export interface HomeConfig extends Config {
   topLevelCommunityList: {
     pageSize: number;
   };
+  /*
+  * Enable or disable the Discover filters on the homepage
+  */
+  showDiscoverFilters: true;
```

Comment:
I think this should say:
```
showDiscoverFilters: boolean;
```

That's how this appears in other `config.interface.ts` files, e.g. [`markdown-config.interface.ts](https://github.com/DSpace/dspace-angular/blob/main/src/config/markdown-config.interface.ts#L14)`